strings_lab.js

```diff
@@ -0,0 +1,46 @@
+// # String Exercises
+//
+// 1. `console.log` the first character of a string.
+let str = "briany"
+console.log(str[0]);
+
+// 2. `console.log` the length of a string.
+console.log(str.length);
+
+// 3. `console.log` the last character of any string.
+console.log(str[str.length - 1]);
+
+// 4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+let name = "thais";
```

Comment:
What would happen if I change name to "Alejandro"? Would this code still give us right answer? Always test your code with different inputs before moving on to next questions. Your code must be able to accommodate to different inputs and still returns consistent and correct outputs.